Automattic/wp-calypso
# Activity: filtering UI tweaks

This PR adds minor UI changes:

## Adds support for dark color scheme and makes buttons behave as expected.

**Before**

![image](https://user-images.githubusercontent.com/390760/45744803-4b555580-bbf7-11e8-9d3a-59fc1bd130a6.png)

**After**

![image](https://user-images.githubusercontent.com/390760/45744820-54462700-bbf7-11e8-966a-ecfe0b540746.png)

![image](https://user-images.githubusercontent.com/390760/45744835-5c9e6200-bbf7-11e8-9ac8-0d36cbdc49fa.png)

## Hides the filter icon in small screens

**Before**

![image](https://user-images.githubusercontent.com/390760/45744891-848dc580-bbf7-11e8-82ce-17f5bb119db3.png)

**After**

![image](https://user-images.githubusercontent.com/390760/45744874-7b045d80-bbf7-11e8-8965-8d24c2d68db8.png)


## How to test

- Run locally or fire up https://calypso.live/?branch=update/activity-log-filtering-ui-tweaks
- Go to Account Settings: `/me/account` and select the Dark color scheme.
- Go to Activity.
- Test:
  - The filtering thoroughly and ensure the button/token labels are always readable.
  - The Activity in a smaller screen and make sure below 480px the icon is removed to make space for the filtering tokens.
